app/services/recording_creator.rb

```diff
@@ -23,6 +23,17 @@ def call
 
   private
 
+  # Returns the visibility of the recording (published, unpublished or protected)
+  def get_recording_visibility(recording:)
+    return 'Protected' if recording[:metadata][:protected] == 'true'
```

Comment:
Uhhhh I don't think `protected` is under `metadata` I think its directly under recording

Reply:
It seems to be metadata in GLv2.

Reply:
Who could confirm this? It's not in the docs.

Reply:
BBB does not support protected yet.
It is in talk for BBB 2.6 or later (later is more likely).